SecondHomework/Logistic/ParcelManager.cs

```diff
@@ -1,134 +1,37 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.Design;
-using System.IO;
-using Newtonsoft.Json;
-
-public class ParcelManager
+﻿public class ParcelManager
 {
-    private const string _filePath = @"C:\Users\kharc\source\repos\SecondHomework\Logistic\Logistic\Parcels.json";
 
-    private List<Parcel> parcels;
+    private List<Parcel> parcels = new List<Parcel>();
+    private FileManager fileManager = new FileManager();
 
     public ParcelManager()
     {
         parcels = new List<Parcel>();
     }
 
-    public void AddParcelFromInput()
-    {
-        Console.Write("What are you planning to send? ");
-        string? name = Console.ReadLine();
-
-        Console.Write("Who is the recipient? ");
-        string? recipient = Console.ReadLine();
-
-        Console.Write("Enter the destination, please: ");
-        string? destination = Console.ReadLine();
-
-        AddParcel(name, recipient, destination);
-    }
-
-    public void Display()
-    {
-        if (parcels.Count == 0)
-        {
-            Logger.NoParcels();
-        }
-        else
-        {
-            foreach (var parcel in parcels)
-            {
-                Console.WriteLine(parcel);
-            }
-        }
-    }
-
-    public void AddParcel(string? name, string? recipient, string? destination)
+    public void AddParcel(Parcel parcel)
     {
-        var parcel = new Parcel(Guid.NewGuid(), name, recipient, destination);
         parcels.Add(parcel);
-        SaveParcels();
     }
 
     public void SaveParcels()
```

Comment:
It should not be public method of ParcelManager, please save data on each add or remove operation